lib/explorer/query.ex

```diff
@@ -292,6 +395,17 @@ defmodule Explorer.Query do
 
   @kernel_only kernel_only -- kernel_only -- kernel_all
 
+  @doc """
+  Builds an `%Explorer.Backend.QueryFrame{}` that returns lazy versions of the
```

Comment:
It returns a regular dataframe and not that struct. We should probably not mention this struct anywhere, except in the implementation detail notes. :)